plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaComputerLauncher.java

```diff
@@ -0,0 +1,149 @@
+        callables.add(new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                ChannelBuilder cb = new ChannelBuilder(computer.getNode().getNodeName(), Computer.threadPoolForRemoting)
+                        .withHeaderStream(listener.getLogger());
+                CommandTransport ct = makeTransport(computer);
+                computer.setChannel(cb, ct, new Channel.Listener() {
+                    @Override
+                    public void onClosed(Channel channel, IOException cause) {
+                        super.onClosed(channel, cause);
+                    }
+                });
+                return true;
+            }
+        });
+        try {
+            long time = System.currentTimeMillis();
+            List<Future<Boolean>> results;
+            final ExecutorService srv = launcherExecutorService;
+            if (srv == null) {
+                throw new IllegalStateException("Launcher Executor Service should be always non-null here, because the task allocates and closes service on its own");
+            }
+            results = srv.invokeAll(callables);
+            Boolean res;
+            try {
+                res = results.get(0).get();
+            } catch (ExecutionException e) {
+                res = Boolean.FALSE;
+            }
+            if (!res) {
+                listener.getLogger().println("Launch failed");
+            } else {
+                System.out.println("Launch successfully");
+            }
+        } catch (InterruptedException e) {
+            System.out.println("Interrupted Exception");
+        } finally {
+            ExecutorService srv = launcherExecutorService;
+            if (srv != null) {
+                srv.shutdownNow();
+                launcherExecutorService = null;
+            }
+        }
+        // testing code.
+//        KafkaProducerClient producer = KafkaProducerClient.getInstance();
+//        KafkaConsumerClient consumer = KafkaConsumerClient.getInstance();
+//        JenkinsLocationConfiguration loc = JenkinsLocationConfiguration.get();
+//        String jenkinsURL = (loc != null && loc.getUrl() != null) ? loc.getUrl() : "http://localhost:8080";
+//        URL url = new URL(jenkinsURL);
+//        String masterAgentConnectionTopic = url.getHost() + "-" + url.getPort() + "-" + computer.getName()
+//                + KafkaConstants.CONNECT_SUFFIX;
+//        String agentMasterConnectionTopic = computer.getName() + "-" + url.getHost() + "-" + url.getPort()
+//                + KafkaConstants.CONNECT_SUFFIX;
+//        Properties producerProps = GlobalKafkaProducerConfiguration.get().getProps();
+//        Properties consumerProps = GlobalKafkaConsumerConfiguration.get().getProps();
+//        producer.send(producerProps, masterAgentConnectionTopic, null, agentMasterConnectionTopic);
+//        consumer.subscribe(consumerProps, Arrays.asList(agentMasterConnectionTopic), 0);
```

Comment:
Again, at some point this unused code should be cleaned up. This is not necessary for this PR.